teeio-validator/library/cxl_ide_test_lib/test_case/ide_km_common.c

```diff
@@ -47,33 +49,37 @@ static bool cxl_ide_generate_key(const void *pci_doe_context,
                                   void *spdm_context, const uint32_t *session_id,
                                   uint8_t stream_id, uint8_t key_sub_stream, uint8_t port_index,
                                   cxl_ide_km_aes_256_gcm_key_buffer_t *key_buffer,
-                                  bool key_iv_gen_capable
+                                  bool key_iv_gen_capable, uint8_t fixed_key_byte
```

Comment:
I dont recommend to have `uint8_t fixed_key_byte`, because that is our design. It may changed in the future, e.g. a uint16 or uint32 fixed key value.

I recommend to add `direction` field to distinguish TX and RX.

In this function, we can decide how to generate the key based on TX and RX.

Reply:
Will update.